execution_workflows/QAPA/bin/make_quant_bed.py

```diff
@@ -9,6 +9,13 @@
 qapa_quant_bed.write('\t'.join(header_fields)+'\n')
```

Comment:
Can you remove this line, please? Thank you!

Reply:
Removed this and leading "chr" in commit 3b8b138499212e8c24d009d3ff83638c2ce4ca00